util/opensslwrap.sh

```diff
@@ -11,13 +11,13 @@ if [ -d "${HERE}../providers" -a "x$OPENSSL_MODULES" = "x" ]; then
 fi
 
 if [ -x "${OPENSSL}.exe" ]; then
-	# The original reason for this script existence is to work around
+	# The original reason for this script's existence is to work around
 	# certain caveats in run-time linker behaviour. On Windows platforms
-	# adjusting $PATH used to be sufficient, but with introduction of
-	# SafeDllSearchMode in XP/2003 the only way to get it right in
+	# adjusting $PATH used to be sufficient, but with the introduction of
+	# SafeDllSearchMode in XP/2003, the only way to get it right in
 	# *all* possible situations is to copy newly built .DLLs to apps/
 	# and test/, which is now done elsewhere... The $PATH is adjusted
-	# for backward compatibility (and nostagical reasons:-).
+	# for backward compatibility (and nostalgic reasons :3).
```

Comment:
The final `:-)` was clearly meant to be a emoticon smile. Either leave as-is or remove completely

Reply:
I did pick up on that. However, it actually blends into the parenthesis, starting before the 'and' in the comment line. Thus, I updated the emoticon slightly so it would persist, while keeping the parenthesis block in tact.

Reply:
Err, to be clear, `nostalgical` was clearly deliberate, but changing to `nostalgic` is fine. It's the change of the emoticon I disagree with here, but happy to see a space added before it.

Reply:
@tom-cosgrove-arm that would be great, but the author didn't type "nostalgical" either :P

Reply:
😁

Reply:
So that we can move forward with this, I suggest the smallest change to the original, just to fix the obvious typo
```suggestion
	# for backward compatibility (and nostalgical reasons:-).
```